Ouch!

```
base_tier_validation/models/tier_definition.py:77:89: E501 Line too long (90 > 88)
```

Please rebase and fix the long line after merging `base_tier_validation`.

--- a/.copier-answers.yml
+++ b/.copier-answers.yml
@@ -1,5 +1,5 @@
 # Do NOT update manually; changes here will be overwritten by Copier
-_commit: v1.19
+_commit: v1.20
 _src_path: gh:oca/oca-addons-repo-template
 additional_ruff_rules: []
 ci: GitHub

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -41,7 +41,7 @@ repos:
     hooks:
       - id: whool-init
   - repo: https://github.com/oca/maintainer-tools
-    rev: f71041f22b8cd68cf7c77b73a14ca8d8cd190a60
+    rev: 9a170331575a265c092ee6b24b845ec508e8ef75
     hooks:
       # update the NOT INSTALLABLE ADDONS section above
       - id: oca-update-pre-commit-excluded-addons
@@ -110,7 +110,7 @@ repos:
         args: [--fix, --exit-non-zero-on-fix]
       - id: ruff-format
   - repo: https://github.com/OCA/pylint-odoo
-    rev: v8.0.19
+    rev: v9.0.4
     hooks:
       - id: pylint_odoo
         name: pylint with optional checks

--- a/.ruff.toml
+++ b/.ruff.toml
@@ -6,6 +6,7 @@ fix = true
 extend-select = [
     "B",
     "C90",
+    "E501",  # line too long (default 88)
     "I",  # isort
     "UP",  # pyupgrade
 ]

--- a/base_tier_validation/models/tier_definition.py
+++ b/base_tier_validation/models/tier_definition.py
@@ -74,7 +74,8 @@ class TierDefinition(models.Model):
         help="Approval order by the specified sequence number",
     )
     approve_sequence_bypass = fields.Boolean(
-        help="Bypassed (auto validated), if previous tier was validated by same reviewer",
+        help="Bypassed (auto validated), if previous tier was validated "
+        "by same reviewer",
     )
 
     @api.onchange("review_type")

